src/frontend.rs

```diff
@@ -746,6 +752,32 @@ impl AsyncComponent for App {
                     }
                 }
             });
+            let input_sender = sender.input_sender().clone();
```

Comment:
Just use `let sender = sender.clone()` and `sender.input()` internally, which has a nicer API that is not fallible.

However, you already have `_sender`, which is designed exactly for this purpose and should be used instead.